internal/cluster/cluster.go

```diff
@@ -153,12 +155,26 @@ func (c *clusterAPI) GarbageCollectBuildsAndSigns(ctx context.Context, mcm hubv1
 	return append(deletedBuilds, deletedSigns...), nil
 }
 
+func (c *clusterAPI) KernelVersions(cluster clusterv1.ManagedCluster) ([]string, error) {
+	for _, clusterClaim := range cluster.Status.ClusterClaims {
+		if clusterClaim.Name != constants.KernelVersionsClusterClaimName {
+			continue
+		}
+
+		kernelVersions := strings.Split(clusterClaim.Value, "\n")
+		sort.Strings(kernelVersions)
+
+		return kernelVersions, nil
+	}
+	return nil, errors.New("KMM ClusterClaim not found")
```

Comment:
```suggestion
	return nil, fmt.Errorf("%s ClusterClaim not found", constants.KernelVersionsClusterClaimName)
```

Reply:
I think the KMM ClusterClaim name can be considered an implementation detail; not sure why we should log it.

Reply:
I agree with @qbarrand that the `ClusterClaim` name is an implementation detail, which we should avoid logging. 

The reason for this is that the user has no control over it. KMM Managed (on the spoke) creates this KMM `ClusterClaim` and this log is there to point users to KMM Managed when they see this error. WDYT?

Reply:
> We can have a bunch of clusterClaims but we return an error only if we didn't find a specific clusterClaim (which is the kernel version).

Exactly. The error message says that we could not find KMM's entry in the cluster claims. I don't think the user needs to know more.

Reply:
Any error message that replaces `ClusterClaim` with `kernel-version ClusterClaim` sounds good to me. I think it will help folks which are not expert in hub&spoke (such as myself when I have written the e2e test) which may have a rough idea of what a clusterClaim is and specifying which kind of clusterClaim helps.

Will you be OK with `kernel-version ClusterClaim not found`?